cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -965,6 +965,12 @@ def ceph_storage():
 
     ceph_admin = endpoint_from_flag('ceph-storage.available')
 
+    # This is needed as we must allow privileged.
+    reconfigure_apiserver(
+        endpoint_from_flag('etcd.available'),
```

Comment:
this flag is not guaranteed to be valid at this point because it isn't in a @when on top of the function. I don't think we could get into here easily without etcd available, but to be safe we should include that in the @when.